File: PageantManager.Business/Entities/Garment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace PageantManager.Business.Entities
{
    public class Garment
    {
        [Key]
        public int GarmentId { get; set; }
        public int GarmentTypeId { get; set; }
        public DateTime AddedDate { get; set; }
        public DateTime? RetiredDate { get; set; }
        public bool CheckedOut { get; set; }
        public string Photo { get; set; }

        public GarmentType GarmentType { get; set; }
        public List<GarmentMeasurement> GarmentMeasurements { get; set; }
    }
}

File: PageantManager.Web/src/app/shared/models/garment.ts
import { GarmentMeasurement } from './garment-measurement';

export class Garment {
  garmentId: number;
  garmentTypeId: number;
  addedDate: Date;
  retiredDate?: Date;
  checkedOut: boolean;
  photo: string;
  garmentMeasurements: GarmentMeasurement[];
}
